input.go

```diff
@@ -18,6 +18,19 @@ import (
 	"github.com/hajimehoshi/ebiten/internal/ui"
 )
 
+// CopyChars copies "printable" runes read from the keyboard each update to buf.
+//
+// CopyChars represents the environment's locale-dependent translation of keyboard
+// input to Unicode characters.
+//
+// IsKeyPressed is based on a mapping of device codes to input device keys.
+// "Control" and modifier keys should be handled with IsKeyPressed.
+//
+// This function is concurrent-safe.
+func CopyChars(buf []rune) int {
```

Comment:
I understand your concern, but as the existing `Touches` returns a copied slice, I prefer the original `InputChars` for consistency. BTW, is there a similar function signature in the standard lib or golang.org/x libs?